emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java

```diff
@@ -30,66 +29,85 @@
 import static com.vanniktech.emoji.Utils.checkNotNull;
 
 public final class EmojiPopup {
-  static final int MIN_KEYBOARD_HEIGHT = 100;
+  private static final int MIN_KEYBOARD_HEIGHT = 100;
```

Comment:
Please leave all of these package-private. They are so that we don't create synthetic accessor methods.